plugins/modules/aci_rest.py

```diff
@@ -537,5 +546,27 @@ def main():
     aci.exit_json(**aci.result)
 
 
+def process_rest_payload(payload, convert_values_to_str):
+    if convert_values_to_str:
+        if isinstance(payload, str):
+            payload = convert_payload_values_to_str(json.loads(payload))
+        else:
+            payload = convert_payload_values_to_str(payload)
+    return payload
+
+
+def convert_payload_values_to_str(data):
+    if data is None:
+        return ""
+    if isinstance(data, dict):
+        return {k: convert_payload_values_to_str(v) for k, v in data.items()}
+    elif isinstance(data, list):
+        return [convert_payload_values_to_str(item) for item in data]
+    elif (isinstance(data, int) and not isinstance(data, bool)) or isinstance(data, float):
```

Comment:
Does this mean only int and float types are converted to strings?
I am curious how the `not bool` condition is used here?

Reply:
In Python, the `bool` type is a subclass of the `int` type. Therefore, to prevent a Boolean from being converted to its string representation ("True" or "False"), we should explicitly check its type using `not isinstance(data, bool)`.